src/metakb/cli.py

```diff
@@ -194,136 +430,225 @@ async def update_metakb_db(
                 msg = f"No valid transform file found matching pattern: {pattern}"
                 raise FileNotFoundError(msg) from e
 
-            g.load_from_json(path)
+            graph.load_from_json(path)
 
-    g.close()
+    graph.close()
     end = timer()
-    echo_info(f"Successfully loaded neo4j database in {(end - start):.5f} s\n")
+    _echo_info(f"Successfully loaded neo4j database in {(end - start):.5f} s")
 
 
-def _help_msg(msg: str = "") -> None:
-    """Handle invalid user input.
+@cli.command()
+@click.option("--db_url", "-u", default="", help=_neo4j_db_url_description)
+@click.option("--db_credentails", "-c", help=_neo4j_creds_description)
+@click.option("--normalizer_db_url", "-n", help=_normalizer_db_url_description)
+@click.option(
+    "--refresh_source_caches",
+    "-r",
+    is_flag=True,
+    default=False,
+    help=(
+        "`True` if source caches (e.g. CivicPy) should be updated prior to data regeneration. Note this will take several minutes. `False` if local cache should be used"
```

Comment:
```suggestion
        "`True` if source caches (e.g. CIViCPy) should be updated prior to data regeneration. Note this will take several minutes. `False` if local cache should be used"
```